file-manager/src/main/scala/it/pagopa/interop/commons/files/StorageConfiguration.scala

```diff
@@ -1,7 +1,11 @@
 package it.pagopa.interop.commons.files
 
-import com.typesafe.config.ConfigFactory
+import com.typesafe.config.{Config, ConfigFactory}
 
 object StorageConfiguration {
-  val maxConcurrency: Int = ConfigFactory.load().getInt("interop-commons.storage.max-concurrency")
+  val config: Config = ConfigFactory.load()
+
+  val maxConcurrency: Int        = config.getInt("interop-commons.storage.max-concurrency")
+  val getUrlDurationMinutes: Int = config.getInt("get-url-duration-minutes")
+  val putUrlDurationMinutes: Int = config.getInt("put-url-duration-minutes")
```

Comment:
These new configurations mean we should add the new environment variables everywhere we use the FileManager.
We can avoid this by not updating the Commons library version except in BFF.

Reply:
mmm it's a very good point, it's not ideal that one service must declare them even when not using the feature. My bad.
What if they become function parameters and we set them directly in the service (BFF)?

Reply:
It seems like a more than reasonable solution to me.